bc/sponsorship/models.py

```diff
@@ -22,10 +22,21 @@ class Sponsorship(AbstractDateTimeModel):
         decimal_places=2,
         default=0,
     )
+    watermark_message = models.CharField(
+        help_text="Short message to include in document's thumbnails",
+        max_length=100,
+        blank=True,
+    )
 
     def __str__(self) -> str:
         return f"{self.pk}: {self.user.username} - {self.original_amount}"
 
+    @property
+    def thumbnail_message(self) -> str:
+        if self.watermark_message:
+            return self.watermark_message
+        return f"This document contributed by {self.user.name}"
```

Comment:
Hm, I think it's worth just requiring the `watermark_message` field as part of every sponsorship. I think sometimes we might want self.user.name, other times, we'd want `self.user.affiliation`, or something else. It's probably just worth thinking about it and not having a default that might be wrong.

Reply:
understood!

Reply:
Sorry, to be more explicit, I think we shouldn't allow blank in the model.

Reply:
got it! Sorry about this. I've removed the `blank` attribute and updated the migration file